components/LessonTabs/LessonTabs.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { CURRICULUM_URL } from '../../constants'
+import NavCard from '../NavCard'
+
+type Lesson = {
+  title: string
+  docUrl?: string | null
+  slug: string
+}
+
+const LessonTabs = ({
+  lesson,
+  activeTab
+}: {
+  lesson: Lesson
+  activeTab: string
+}) => {
+  const tabs = [
+    ...(lesson.docUrl ? [{ text: 'lesson', url: lesson.docUrl }] : []),
+    { text: 'challenges', url: `/curriculum/${lesson.slug}` },
+    { text: 'exercises', url: `/exercises/${lesson.slug}` },
```

Comment:
Might as well add this path to the constants.

Reply:
The exercises shouldn't be under `/exercises` but something like `/curriculum/js0/exercises`.

Reply:
Yes, i also think the mentor path also needs a bit more thought!

Reply:
> The exercises shouldn't be under /exercises but something like /curriculum/js0/exercises.

@flacial Can you explain the reason why the above should mean that we shouldn't make a `/exercises` constant?

Reply:
> I thought /exercises/js0 would make things inconsistent with the way we manage things that are related to a lesson (e.g, challenges) and thus it should be changed to something like /curriculum/js0/exercises. 

I'm not understanding the reasoning -- even if the path is `/curriculum/js0/exercises`, why do you think a `/exercises` constant should still not be made for that?

Reply:
> > I thought /exercises/js0 would make things inconsistent with the way we manage things that are related to a lesson (e.g, challenges) and thus it should be changed to something like /curriculum/js0/exercises.
> 
> I'm not understanding the reasoning -- even if the path is `/curriculum/js0/exercises`, why do you think a `/exercises` constant should still not be made for that?

I decided not to add that constant because, based on the possibility of introducing inconsistency in how we manage things that are related to a lesson, we could change the exercises' paths before launching DOJO. I was wrong as we already have the `/review/[lessonSlug]` path.

Reply:
I was thinking that since we are going to need `/exercises` in the path it would be a good idea to make the constant anyway(which I see you note you'll do in another PR). Whether the path is `/curriculum/js0/exercises` or `/exercises/js0` it needs `/exercises`. The reason I think `/exercises` should be a constant is because of its high potential for reusability -- if you search the path `/curriculum` for example, you can find a lot of results of its usage,
 https://github.com/garageScript/c0d3-app/search?p=4&q=%2Fcurriculum

Reply:
I like this idea. By creating the exercises constant, we can join constants to create paths like the following `CURRICULUM_PATH/[lessonSlug]/REVIEW_PATh`.